react/features/base/settings/middleware.js

```diff
@@ -115,6 +122,25 @@ function _maybeSetAudioOnly(
     }
 }
 
+/**
+ * Updates the display name to the one in JWT for vpaas meetings.
+ *
+ * @param {Store} store - The redux store.
+ * @private
+ * @returns {void}
+ */
+function _maybeUpdateDisplayName({ dispatch, getState }) {
+    const state = getState();
+
+    if (isVpaasMeeting(state)) {
```

Comment:
Is there a reason to limit this only for vpass? Can we use the jwt name in general if it exist. Can we make it more generic, is what I mean?

Reply:
Good point! I'll check the shape of the jwt and make it more generic.

Reply:
Yep name is in the general jwt structure as described here: https://github.com/jitsi/lib-jitsi-meet/blob/master/doc/tokens.md#payload

Reply:
Done.